db/seeds/development.rb

```diff
@@ -40,3 +39,23 @@
                access_token_audience: 'va.gov',
                logout_redirect_uri: 'http://localhost:3001',
                refresh_token_duration: SignIn::Constants::RefreshToken::VALIDITY_LENGTH_SHORT_MINUTES)
+
+# Create Config for example external client using cookie auth
+cookie_template = SignIn::ClientConfig.find_or_initialize_by(client_id: 'cookie_template')
```

Comment:
Could we make the client_ids just a bit more descriptive? Wonder if it could be something like:
`sample_client_web`, `sample_client_api`
Could be another name if something else is better too, just want to make this clearly differentiated from the `va` client ids